NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj

```diff
@@ -25,9 +25,10 @@
     <MtouchArch>x86_64</MtouchArch>
     <MtouchLink>None</MtouchLink>
     <MtouchDebug>true</MtouchDebug>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>Apple Development: Antoni Kistowski (TZ2K58WDRV)</CodesignKey>
```

Comment:
probably you have something wrong in project configuration if it adds your name here. but it is a minor issue

Reply:
Okey, already changed this project configuration.